### DISCUSSION TITLE
How Create C# Console Application that will listen to broker until it is closed manually.

### DISCUSSION THREAD

**Discussion by yesh-rajawat-robomq** [rabbitmq/rabbitmq-dotnet-client#1511] [Q&A]
How can I create an console application that will listen to the broker infinitely and if there occurs any error how do i handle it, please help

**Comment by lukebakken** [comment]
Hello, thanks for using RabbitMQ and this library.

We are not a code-writing service. I am happy to assist you, but you must first give it a try yourself. If you are stuck, share your code in a git repository (via GitHub, for instance), and I will see what I can do.

**Comment by michaelklishin** [comment]
@yesh-rajawat-robomq the open source community can answer specific questions but we won't do your job (or a school assignment) for you.

Unless you have an existing code snippet with a reasonably specific question, we won't be able to help.
